python/cudf/CMakeLists.txt

```diff
@@ -14,10 +14,22 @@
 
 cmake_minimum_required(VERSION 3.23.1 FATAL_ERROR)
 
-set(cudf_version 22.10.00)
+if(POLICY CMP0094)
+  # Make FindPython{,2,3} modules use LOCATION for lookup strategy.
+  # https://cmake.org/cmake/help/latest/policy/CMP0094.html#policy:CMP0094
+  cmake_policy(SET CMP0094 NEW)
+  set(CMAKE_POLICY_DEFAULT_CMP0094 NEW)
+endif()
 
 include(../../fetch_rapids.cmake)
 
+include(rapids-cuda)
+# TODO: This will not be necessary once we upgrade to CMake 3.22, which will pull in the required
```

Comment:
This should no longer be needed. https://github.com/rapidsai/cudf/pull/11751 requires CMake 3.23.1 or newer.